src/senaite/jsonapi/fieldmanagers.py

```diff
@@ -504,3 +510,20 @@ def json_data(self, instance, default=[]):
         value = self.get(instance)
         out = map(api.get_url_info, value)
         return out or default
+
+    def set(self, instance, value, **kw):
+        """
+        The Setter!
+        :param instance:
+        :param value: list of dictionaries where AS UID can be found.
+        :param kw:
+        :return:
+        """
+
+        assert type(value) in (list, tuple)
```

Comment:
Please return a more descriptive error message to avoid just a raw `AssertionError`.

Better use:

```python
if not isinstance(value, (list, tuple)):
    raise ValueError("Value must be a list")
```